packages/calling/calling/src/CallingClient/metrics/index.ts

```diff
@@ -262,10 +262,12 @@ class MetricManager implements IMetricManager {
  * @param webex - Webex object to communicate with metrics microservice.
  * @param indicator - Service Indicator.
  */
-export const getMetricManager = (webex: WebexSDK, indicator: ServiceIndicator): IMetricManager => {
```

Comment:
Why was this change required

Reply:
fixed a eslint warning which says that every file should have a default export.